docs/api.md

```diff
@@ -362,6 +363,11 @@ Disconnects Puppeteer from the browser, but leaves the Chromium process running.
 #### browser.targets()
 - returns: <[Array]<[Target]>> An array of all active targets.
 
+#### browser.userAgent()
+- returns: <[Promise]<[string]>> Promise which resolves to the browser's original user agent. For headless Chromium, this includes a string like `HeadlessChrome/61.0.3153.0`.
+
+> **NOTE** browser.userAgent() does not always match the actual user agent if it's overridden by page.setUserAgent(userAgent) or page.emulate(options).
```

Comment:
Maybe we can create `page.userAgent()` to return the actual user agent. But in that case the user must have enough control on which user agent to send, so I don't find necessity to do that.

Reply:
Your comment makes me think that you don't need the browser's user agent, but instead you're interested in the page's user agent. Is this the case?

Reply:
Not exactly.

In my case, I need to get browser's user agent even before opening a new page.

I'm writing a crawler based on Puppeteer.
The basic logic is here:

1. Access the target sites' robots.txt
2. Check robots.txt to see if the user agent is allowed
3. If the user agent is not allowed, stop here and don't go to the next step.
4. Open a new page by `browser.newPage()`
5. Start scraping the page

The code is [here](https://github.com/yujiosaka/headless-chrome-crawler/blob/master/lib/hccrawler.js#L302).

Reply:
Thanks, I understand now.

nit: let's change the "NOTE" text to the following:

> **NOTE** Pages can override browser user agent with [`page.setUserAgent`](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagesetuseragentuseragent)

Reply:
Done
https://github.com/GoogleChrome/puppeteer/pull/1716/commits/5565ed6c76b1926fedd031df06c9e1f824b3f5d7